docs/tutorials/query-language.md

```diff
@@ -0,0 +1,58 @@
+The Realm JavaScript SDK supports querying based on a language inspired by [NSPredicate](https://realm.io/news/nspredicate-cheatsheet/). 
+
+You query your Realm using the {@link Realm.Collection#filtered Collection.filtered()} method:
+
+```JS
+let dogs = realm.objects('Dog');
```

Comment:
I really hate that we use dogs i all of our examples. I take partial responsibility for this. Maybe replace with Apps or Customers?